src/sidebar/store/modules/annotations.js

```diff
@@ -437,6 +442,25 @@ function findAnnotationByID(state, id) {
   return findByID(state.annotations.annotations, id);
 }
 
+/**
+ * Return all loaded annotations which have not been saved to the server.
+ */
+function newAnnotations(state) {
```

Comment:
Not actually used, but seemed like a natural analog to `newHighlights`. Can kill it if it feels extra/not useful.